### PR TITLE
🐛 passing data from a service to another fails

### PR DESCRIPTION
**Pull request by sanderegg**
## What do these changes do?
- @elisabettai found out that it was not possible to connect 2 services together anymore. the /inputs:match endpoint of the webserver returns 422 (e.g. 2 sleepers could not be connected anymore)

This PR fixes this problem

### CODE REVIEW

**Review by elisabettai — APPROVED** [review]
Many thanks for the quick fix!

File: services/web/server/src/simcore_service_webserver/catalog/_handlers.py [hunk 1]
```diff
@@ -74,7 +74,7 @@ def create(cls, request: Request) -> "_RequestContext":
 
 
 class _ServicePathParams(BaseModel):
-    service_key: ServiceKeyEncoded
+    service_key: ServiceKey
```
Comment (GitHK) [inline services/web/server/src/simcore_service_webserver/catalog/_handlers.py:77]: interesting one

**Review by GitHK — APPROVED** [review]

**Review by matusdrobuliak66 — APPROVED** [review]

File: services/web/server/src/simcore_service_webserver/catalog/_handlers.py (hunk 1 above)
Comment (pcrespov) [inline services/web/server/src/simcore_service_webserver/catalog/_handlers.py:77]: As expected, this change will break the tests. 
I think you were right when claimed that our test client do not represent the actual client calls from the front-end

A fast way out of this issue (until we have the time to refactor it properly) is to define here ``service_key: str``  which will "disable" the validation at the handler level. This is effectively as it was before (i.e. no validation whatsoever!) ...

Comment (sanderegg) [reply services/web/server/src/simcore_service_webserver/catalog/_handlers.py:78]: ok I fixed it by always unquoting. this makes the tests pass and does not break the production neither. I think this must be something weird with the aiohttp_server fixture??

**Pull request merged**